Ajout de l'activité Encodage en Ville

Carson-9 opened the pull request.

Carson-9 committed:
Ajout de l'activité "Pixels"

Carson-9 committed:
Update and rename howto.md to readme.md

Carson-9 committed:
Ajout du compte rendu

Carson-9 committed:
Ajout des fichiers source .tex & Nom du folder CompteRendu

Carson-9 committed:
Delete Pixels/ComteRendu directory

Carson-9 committed:
Ajout de l'activité Encodage En Ville

Carson-9 committed:
Merge branch 'InfoSansOrdi:trunk' into trunk

Carson-9 committed:
Ajout de l'activité Encodage en Ville

mquinson commented:
Bonjour,
pourriez-vous mettre le source des cartes de ville s'il vous plait ? Le compte-rendu d'activité devrait être dans un fichier séparé en markdown alors que j'ai l'impression qu'il est à la fin de la fiche de prep et en LaTeX.
En tout cas, c'est du super boulot, bravo.

Carson-9 committed:
Create graph_source.md

Carson-9 commented:
Bonjour,
Je pensais vous croiser mardi, mais je n'en ai pas eu l'occasion. J'ai ajouté un fichier ""détaillant"" la source des graphes : Ces derniers ont été générés sur un outil en ligne dont le lien se trouve dans "graph_source.md".
En ce qui concerne le compte-rendu, tout comme pour la première activité, nous avons créé un dossier "CompteRendu" dans lequel se trouve le compte rendu en LaTeX. Voulez-vous que je transforme ce fichier .tex en .md?

mquinson commented:
Bonjour, oui, le mieux serait d'avoir le rapport d'étonnement en markdown s'il te plait. Au pire, je m'en chargerais, mais c'est encore mieux si tu veux bien le faire.

Carson-9 committed:
Ajout du compte-rendu en markdown

Carson-9 committed:
Update 2025_HasleyKazem_CompteRendu.md

Carson-9 committed:
Ajout d'une version README du compte-rendu.

Stackoverflow me dit qu'il est possible de créer un fichier .gitattribute à la racine du projet pour pouvoir plus ou moins hotswap les fichiers markdown affichés par github dans chaque folder. Piste à suivre?

Carson-9 commented:
Bonjour,
Je viens d'ajouter une version Markdown dans le même dossier, ainsi qu'une copie en README.

mquinson commented:
Merci du super boulot. Un peu dommage que les cartes soient au format bitmap et non vectoriel, c'est un peu moins pratique à éditer pour les prochains. Mais le travail reste impressionnant, bravo.
